import unittest
from exceptions import UserWarning

class utilTest(unittest.TestCase):
    def test_all(self, ):
        raise UserWarning("And now for something completely different... implement this test")